exporter/elasticsearchexporter/README.md

```diff
@@ -16,13 +16,75 @@ This exporter supports sending OpenTelemetry logs and traces to [Elasticsearch](
-  ELASTICSEARCH_URL environment variable will be used.
-- `cloudid` (optional):
-  [ID](https://www.elastic.co/guide/en/cloud/current/ec-cloud-id.html) of the
-  Elastic Cloud Cluster to publish events to. The `cloudid` can be used instead
-  of `endpoints`.
-- `num_workers` (default=runtime.NumCPU()): Number of workers publishing bulk requests concurrently.
+Exactly one of the following settings is required:
+
+- `endpoint` (no default): The target Elasticsearch URL to which data will be sent
+  (e.g. `https://elasticsearch:9200`)
+- `endpoints` (no default): A list of Elasticsearch URLs to which data will be sent
+- `cloudid` (no default): The [Elastic Cloud ID](https://www.elastic.co/guide/en/cloud/current/ec-cloud-id.html)
+  of the Elastic Cloud Cluster to which data will be sent (e.g. `foo:YmFyLmNsb3VkLmVzLmlvJGFiYzEyMyRkZWY0NTY=`)
+
+If none of the above attributes are specified, one or more Elasticsearch URLs
+may be specified as a comma-separated list via the `ELASTICSEARCH_URL`
+environment variable. When a list of endpoints is configured, requests will
+be sent to them in round-robin order for failover.
+
+Elasticsearch credentials may be configured via [Authentication configuration](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configauth/README.md#authentication-configuration) (`configauth`) settings.
+As a shortcut, the following settings are also supported:
+
+- `user` (optional): Username used for HTTP Basic Authentication.
+- `password` (optional): Password used for HTTP Basic Authentication.
+- `api_key` (optional):  Authorization [API Key](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-create-api-key.html) in "encoded" format.
+
+Example:
+
+```yaml
+exporters:
+  elasticsearch:
+    endpoint: https://elastic.example.com:9200
+    auth:
+      authenticator: basicauth
+
+extensions:
+  basicauth:
+    username: elastic
+    password: changeme
+
+······
+
+service:
+  extensions: [basicauth]
+  pipelines:
+    logs:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [elasticsearch/log]
+    traces:
+      receivers: [otlp]
+      exporters: [elasticsearch/trace]
+      processors: [batch]
+```
+
+## Advanced configuration
+
+### HTTP settings
+
+The Elasticsearch exporter supports common [HTTP Configuration Settings](https://github.com/open-telemetry/opentelemetry-collector/tree/main/config/confighttp#http-configuration-settings), except for `compression` (all requests are uncompressed).
+As a consequence of supporting `confighttp`, the Elasticsearch exporter also supports common [TLS Configuration Settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md#tls-configuration-settings).
+
+The Elasticsearch exporter sets `timeout` (HTTP request timeout) to 90s by default.
+All other defaults are as defined by `confighttp`.
```

Comment:
I was trying to stick to linking once on first reference (though to fair I didn't actually use the `confighttp` ref above). I'll play with it and see if it's too many links, or otherwise reference properly above.